maquetado/scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+    color:blue ;
+}
```

Comment:
Scss and htmls should end with an empty lines, but dont worry. Once in the react training, you will have a linter that will warn you.